build/tools/psd1-to-nuspec.ps1

```diff
@@ -0,0 +1,564 @@
+#Requires -Modules PowerShellGet
+#Requires -Version 5.0
+<#
+    =======================================================================================================
+    AUTHOR:  Tao Yang 
```

Comment:
😏

Reply:
Maybe add a reference to the location where this was copied from in the comment?

Reply:
Ok.